website/tables/aws/aws_elbv2_listener_rules.md

```diff
@@ -0,0 +1,28 @@
+# Table: aws_elbv2_listener_rules
+
+This table shows data for Amazon Elastic Load Balancer (ELB) v2 Listener Rules.
+
+https://docs.aws.amazon.com/elasticloadbalancing/latest/APIReference/API_Rule.html
+
+The primary key for this table is **rule_arn**.
+
+## Relations
+
+This table depends on [aws_elbv2_listeners](aws_elbv2_listeners).
+
+## Columns
+
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_source_name|String|
+|_cq_sync_time|Timestamp|
+|_cq_id|UUID|
+|_cq_parent_id|UUID|
+|account_id|String|
+|region|String|
+|listener_arn|String|
+|actions|JSON|
+|conditions|JSON|
+|is_default|Bool|
+|priority|String|
+|rule_arn (PK)|String|
```

Comment:
I think we should add an `arn` column (alias of `rule_arn`) and make that the PK instead, for consistency and so that the AWS resources view will work for this (it assumes the resource ARN is in a column called `arn`).

Reply:
good catch I'll open a follow up PR